src/components/bulk-email-tool/bulk-email-form/messages.js

```diff
@@ -0,0 +1,89 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
```

Comment:
Thanks for doing this! I think it makes the component source code much easier to read with these resourced in a dedicated file.